website/src/tutorials/text-salience.md

```diff
@@ -0,0 +1,318 @@
+  Intuitively, a positive influence score (attribution) for a token (or word,
+  depending on your method) in an example means that if this token was removed
+  we expect a drop in model confidence in the prediction of the class.
+  Similarly, removing a negative token would correspond to an increase in the
+  model's confidence in the prediction of this class."%}
+
+{% include partials/inset-image image: '/assets/images/text-salience-image-2.png',
+  caption: 'TODO'%}
+
+#### Token-Based Methods
+
+[Gradient saturation](https://towardsdatascience.com/the-vanishing-gradient-problem-69bf08b15484)
+is a potential problem for all of the Gradient based methods, such as [Grad L2 Norm](https://aclanthology.org/P18-1032/)
+and [Grad · Input](https://arxiv.org/abs/1412.6815), that we need to look out
+for. Essentially if the model learning saturates for a particular token, then
+its gradient goes to zero and appears to have zero salience. At the same time,
+some tokens actually have a zero salience score, because they do not affect the
+predictions. And there is no simple way to tell if a token that we are
+interested in is legitimately irrelevant or if we are just observing the effects
+of gradient saturation.
+
+The [integrated gradients](https://arxiv.org/pdf/1703.01365.pdf) method
+addresses the gradient saturation problem by enriching gradients with
+embeddings.
+[Tokens](https://jalammar.github.io/a-visual-guide-to-using-bert-for-the-first-time/)
+are the discrete building blocks of text sequences, but they can also be
+represented as vectors in a [continuous embedding space](https://colah.github.io/posts/2014-07-NLP-RNNs-Representations/).
+IG computes per-token salience as the average salience over a set of local
+gradients computed by interpolating between the token’s embedding vectors and a
+baseline (typically the zero vector). The tradeoff is that IG requires more
+effort to identify the right number of interpolation steps to be
+effective (configurable in LIT’s interface), with the number of steps
+correlating directly with runtime. It also requires more information,
+which the model may or may not be able to provide.
+
+{% include partials/inset-image image: '/assets/images/text-salience-image-3.png',
+  caption: 'TODO'%}
+
+#### Blackbox Methods
+
+Some models do not provide tokens or token-level gradients, effectively making
+them blackboxes. [LIME](https://arxiv.org/pdf/1602.04938v3.pdf) can be used with
+these models. LIME works by generating a set of perturbed inputs, generally, by
+dropping out or masking tokens, and training a local linear model to reconstruct
+the original model's predictions. The weights of this linear model are treated
+as the salience values.
+
+LIME has two limitations, compared to gradient-based methods: it can be slow as
+it requires many evaluations of the model, and it can be noisy on longer inputs
+where there are more tokens to ablate. We can increase **_the number of samples to be used for LIME_**
```

Comment:
```suggestion
LIME has two limitations, compared to gradient-based methods: 

1.  it can be slow as it requires many evaluations of the model, and
2.  it can be noisy on longer inputs where there are more tokens to ablate. 

We can increase **_the number of samples to be used for LIME_**
```